src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs.rs

```diff
@@ -0,0 +1,363 @@
+//! This module provides a view for breadcrumbs, enabling us to know which node the graph being
+//! edited belongs to and navigating through them.
+
+use crate::prelude::*;
+
+pub mod breadcrumb;
+pub mod project_name;
+
+pub use breadcrumb::Breadcrumb;
+pub use project_name::ProjectName;
+
+use crate::graph_editor::MethodPointer;
```

Comment:
I don't think we have any rule for separating pub use declarations, so they likely should go together with non-pub ones, with crate imports leading.

(though it might be worth discussing at some point, as I doin't really like having pub and non-pub use decls mixed up)

Reply:
I don't like mixing re-exports with regular use either. Can't we just leave it as it is? I think it's more readable.

Reply:
Personally I'm find but the rule should be adjusted.
@farmaazon  what's your stance?

Reply:
`pub use` should be separated and should be placed above all other imports IMO. This rule is clearly missing.